build.sbt

```diff
@@ -245,6 +245,11 @@ lazy val cliTest = project
     testkit
   )
 
+lazy val dlcSuredbitsClient = project
```

Comment:
Make sure you add this to `aggregate()` on `bitcoin-s` so tasks run at the root level are run in the subproject too: https://www.scala-sbt.org/1.x/docs/Multi-Project.html#Aggregation